rest/rest-server/src/main/java/org/ow2/proactive_grid_cloud_portal/scheduler/SchedulerStateRest.java

```diff
@@ -2299,6 +2276,34 @@ public JobIdData submitFromUrl(@HeaderParam("sessionid") String sessionId, @Head
     public JobIdData submit(@HeaderParam("sessionid") String sessionId, @PathParam("path") PathSegment pathSegment,
             MultipartFormDataInput multipart) throws JobCreationRestException, NotConnectedRestException,
             PermissionRestException, SubmissionClosedRestException, IOException {
+        return submit(sessionId, pathSegment, multipart, null);
+    }
+
+    /**
+     * Submits a job to the scheduler
+     *
+     * @param sessionId
+     *            a valid session id
+     * @param pathSegment
+     *            variables of the workflow
+     * @param multipart
+     *            the form data containing : - fileName the name of the file
+     *            that will be created on the DataSpace - fileContent the
+     *            content of the file
+     * @param contextInfos
+     *            the context informations (generic parameters,..)
+     * @return the <code>jobid</code> of the newly created job
+     * @throws IOException
+     *             if the job was not correctly uploaded/stored
+     */
+    @Override
+    @POST
+    @Path("{path:submit}")
+    @Consumes(MediaType.MULTIPART_FORM_DATA)
+    @Produces("application/json")
+    public JobIdData submit(@HeaderParam("sessionid") String sessionId, @PathParam("path") PathSegment pathSegment,
+            MultipartFormDataInput multipart, @Context UriInfo contextInfos) throws JobCreationRestException,
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove the redundant '!unknownSymbol!' thrown exception declaration(s). [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3ARedundantThrowsDeclarationCheck)